src/plugins/template/tests/functional/op_eval/memory.cpp

```diff
@@ -21,14 +21,6 @@ using namespace ngraph::opset7;
 
 OPENVINO_SUPPRESS_DEPRECATED_START
```

Comment:
Why do we still have this file? I thought that all tests should be moved to: https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/template/tests/functional/op_reference